doc/swish/parallel.tex

```diff
@@ -0,0 +1,183 @@
+% Copyright 2023 Beckman Coulter, Inc.
```

Comment:
2024? Maybe now is a good time to change the 2023 to 2024 in doc/swish.tex?

Reply:
I explicitly opted for 2023 because the code was created and published to a public fork in 2023.  I will add a separate commit to update doc/swish.tex.